### PR TITLE
Improve the summary of the test report

### PR DESCRIPTION
**Pull request by Poor12**
Signed-off-by: Poor12 <shentiecheng@huawei.com>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
None

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/assign @RainbowMango

File: i18n/zh/docusaurus-plugin-content-blog/2022-10-26-test-report/index.md [hunk 1]
```diff
@@ -1,10 +1,16 @@
 ---
-title: 突破100集群和500,000节点，Karmada大规模测试报告发布
+title: 突破100倍集群规模，Karmada大规模测试报告发布
 ---
 
 ## 摘要
 
-随着Karmada在越来越多的企业和组织中的大规模落地，Karmada的可扩展性和大规模逐渐成为社区用户的新关注点。因此，我们对Karmada开展了大规模测试工作，根据测试结果分析，以Karmada为核心的集群联邦可以稳定支持100个集群和50万个节点同时在线，管理超过200万个Pod。在本文中，我们将介绍用于测试的相关指标，如何进行大规模测试，以及我们如何实现大规模节点和集群接入。
+随着云原生技术在越来越多的企业和组织中的大规模落地，如何高效、可靠地管理大规模的资源池以应对不断增长的业务挑战成为了当下云原生技术的关键挑战。
```
Comment (RainbowMango) [inline i18n/zh/docusaurus-plugin-content-blog/2022-10-26-test-report/index.md:7]:
```suggestion
随着云原生技术在越来越多的企业和组织中的大规模落地，如何高效、可靠地管理大规模资源池以应对不断增长的业务挑战成为了当下云原生技术的关键挑战。
```

File: i18n/zh/docusaurus-plugin-content-blog/2022-10-26-test-report/index.md [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
+在过去的很长一段时间内，不同厂商尝试通过定制Kubernetes原生组件的方式扩展单集群的规模，这在提高规模的同时也引入了复杂的单集群运维、不清晰的集群升级路径等问题。
+而多集群技术能在不侵入式地修改Kubernetes单集群的基础上横向扩展资源池的规模，在扩展资源池的同时降低了企业的运维管理等成本。
```
Comment (RainbowMango) [inline i18n/zh/docusaurus-plugin-content-blog/2022-10-26-test-report/index.md:9]:
```suggestion
而多集群技术能在不侵入修改Kubernetes单集群的基础上横向扩展资源池的规模，在扩展资源池的同时降低了企业的运维管理等成本。
```

File: i18n/zh/docusaurus-plugin-content-blog/2022-10-26-test-report/index.md [hunk 3]
```diff
@@ -1,10 +1,21 @@
...
 ## 摘要
 
-随着Karmada在越来越多的企业和组织中的大规模落地，Karmada的可扩展性和大规模逐渐成为社区用户的新关注点。因此，我们对Karmada开展了大规模测试工作，根据测试结果分析，以Karmada为核心的集群联邦可以稳定支持100个集群和50万个节点同时在线，管理超过200万个Pod。在本文中，我们将介绍用于测试的相关指标，如何进行大规模测试，以及我们如何实现大规模节点和集群接入。
+随着云原生技术在越来越多的企业和组织中的大规模落地，如何高效、可靠地管理大规模资源池以应对不断增长的业务挑战成为了当下云原生技术的关键挑战。
+在过去的很长一段时间内，不同厂商尝试通过定制Kubernetes原生组件的方式扩展单集群的规模，这在扩大规模的同时也引入了复杂的单集群运维、不清晰的集群升级路径等问题。
+而多集群技术能在不侵入修改Kubernetes单集群的基础上横向扩展资源池的规模，在扩展资源池的同时降低了企业的运维管理等成本。
+
+在Karmada的大规模落地进程中，Karmada的可扩展性和大规模逐渐成为社区用户的新关注点。
+因此，我们对Karmada开展了大规模环境下的测试工作，以获取Karmada管理多个Kubernetes集群的性能基线指标。
+**对于以Karmada为代表的多集群系统而言，单集群的规模不是制约它的资源池规模的限制因素。**
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-blog/2022-10-26-test-report/index.md:13]:
```suggestion

**对于以Karmada为代表的多集群系统而言，单集群的5k节点上限不再是制约资源池规模扩张的限制因素。**
```

File: i18n/zh/docusaurus-plugin-content-blog/2022-10-26-test-report/index.md [hunk 4]
```diff
@@ -1,10 +1,21 @@
...
+因此，我们参考了**Kubernetes的大规模集群的标准配置和用户的生产落地实践**，测试了Karmada同时管理100个5k节点和2wPod的Kubernetes集群的用户场景。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-blog/2022-10-26-test-report/index.md:14]:
```suggestion
我们参考了**Kubernetes大规模集群的标准配置和用户的生产落地实践**，测试了Karmada同时管理100个5k节点和100个2万Pod构成的Kubernetes集群使用场景。
```

File: i18n/zh/docusaurus-plugin-content-blog/2022-10-26-test-report/index.md [hunk 5]
```diff
@@ -1,10 +1,21 @@
...
+受限于测试环境和测试工具，本次测试并未追求测试到Karmada多集群系统的上限，而是希望能覆盖到在生产中大规模使用多集群技术的典型场景。
+根据测试结果分析，以Karmada为核心的集群联邦可以**稳定支持100个大规模集群**，管理超过50万个节点和200万个Pod，可以满足用户在大规模生产落地的需要。
+
+在本文中，我们将介绍用于测试的相关指标，如何进行大规模测试，以及我们如何实现大规模的集群接入。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-blog/2022-10-26-test-report/index.md:18]:
```suggestion
在本文中，我们将介绍测试相关指标、大规模测试以及大规模集群接入方法。
```

**Review by windsonsea — COMMENTED** [review]
wow, so many words! 

cool, I have read the graph and text :)

File: i18n/zh/docusaurus-plugin-content-blog/2022-10-26-test-report/index.md (hunk 4 above)
Comment (Poor12) [reply i18n/zh/docusaurus-plugin-content-blog/2022-10-26-test-report/index.md:14]: The following modification would be misinterpreted as having 200 clusters.

File: i18n/zh/docusaurus-plugin-content-blog/2022-10-26-test-report/index.md (hunk 3 above)
Comment (Poor12) [reply i18n/zh/docusaurus-plugin-content-blog/2022-10-26-test-report/index.md:13]: The scale here refers not only to nodes, but also to the number of Pods, the number of namespaces, and so on.

File: i18n/zh/docusaurus-plugin-content-blog/2022-10-26-test-report/index.md (hunk 4 above)
Comment (windsonsea) [reply i18n/zh/docusaurus-plugin-content-blog/2022-10-26-test-report/index.md:14]: All right, only concern 2w may be not an official unit of number 😺

Comment (Poor12) [reply i18n/zh/docusaurus-plugin-content-blog/2022-10-26-test-report/index.md:14]: Mainly limited by `kind`. A simulated cluster cannot run so many Pods.

**Comment by RainbowMango** [comment]
Please @Tingtal take a look. This might be posted on CNCF blog.

File: blog/2022-10-26-test-report/index.md [hunk 6]
```diff
@@ -410,7 +421,7 @@ Karmada-agent in the member cluster with 5k Nodes and 2w Pods consumes 40m CPU(c
 ## Conclusion and Analysis
 
 From the preceding test results, the API Call Latency and Resource Distribution Latency meet the SLIs/SLOs above, and the resources consumed by the system during the whole process are in a controllable range.
-Therefore, **Karmada can stably support 100 clusters with 500,000 nodes and more than two million pods**. In production, Karmada effectively supports the management of hundreds of large-sized clusters.
+Therefore, Karmada can **stably support 100 large-scale clusters**, manage more than 500,000 nodes and 2 million Pods. In production, Karmada effectively supports the management of hundreds of large-sized clusters.
```
Comment (Tingtal) [inline blog/2022-10-26-test-report/index.md:424]:
```suggestion
As tested, Karmada can **stably support 100 large-scale clusters** running more than 500,000 nodes and 2 million pods, and hundreds of such clusters in production environments.
```

File: blog/2022-10-26-test-report/index.md [hunk 7]
```diff
@@ -460,9 +471,14 @@ Now Karmada provides the ability named [cluster resource modeling](https://karma
 In the process of resource modeling, it will collect node and pod information from all clusters managed by Karmada. This imposes a considerable performance burden in large-scale scenarios.
 If you do not use this ability, you can [disable cluster resource modeling](https://karmada.io/docs/next/userguide/scheduling/cluster-resources#disable-cluster-resource-modeling) to further reduce resource consumption.
 
-### Summary
+### Summary and Outlook
+
+According to the analysis of test results, Karmada can **stably support 100 large-scale clusters**, manage more than 500,000 nodes and 2 million Pods.
```
Comment (Tingtal) [inline blog/2022-10-26-test-report/index.md:476]:
```suggestion
As tested, Karmada can **stably support 100 large-scale clusters** running more than 500,000 nodes and 2 million pods.
```

File: blog/2022-10-26-test-report/index.md [hunk 8]
```diff
@@ -460,9 +471,14 @@ Now Karmada provides the ability named [cluster resource modeling](https://karma
...
 
 In terms of use scenarios, Push mode is suitable for managing Kubernetes clusters on public clouds, while Pull mode covers private cloud and edge-related scenarios. In terms of performance and security, the Pull mode outperforms the Push mode.
 Each cluster is managed by the karmada-agent component in the cluster and is completely isolated.
 However, while the Pull mode improves performance, it also needs to correspondingly improve the performance of karmada-apiserver and karmada-etcd to face challenges in traffic-intensive and high-concurrency scenarios. For specific methods, please refer to the optimization of Kubernetes for large-scale clusters.
 In general, users can choose different deployment modes according to usage scenarios to improve the performance of the entire multi-cluster system through parameter tuning and other means.
+
+Limited by the test environment and test tools, this test has not yet tested the upper limit of Karmada. And the performance test of multi-cluster system is still in the ascendant stage.
```
Comment (Tingtal) [inline blog/2022-10-26-test-report/index.md:483]:
```suggestion
Limited by the environment and tooling, this test does not stress test Karmada. The industry just got started on performance tests of multi-cluster systems.
```

File: blog/2022-10-26-test-report/index.md [hunk 9]
```diff
@@ -460,9 +471,14 @@ Now Karmada provides the ability named [cluster resource modeling](https://karma
...
+In the next step, we will continue to optimize the test tools of multi-cluster system and systematically organize the test methods to cover large scales and more scenarios.
```
Comment (Tingtal) [inline blog/2022-10-26-test-report/index.md:484]:
```suggestion
Our next steps will be taken on better tools and methodologies for testing multi-cluster systems in more scenarios.
```

**Review by Tingtal — CHANGES_REQUESTED** [review]
Language checks.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**